modules/monitoring-setting-scrape-sample-and-label-limits-for-user-defined-projects.adoc

```diff
@@ -44,7 +45,33 @@ data:
 ----
 <1> A value is required if this parameter is specified. This `enforcedSampleLimit` example limits the number of samples that can be accepted per target scrape in user-defined projects to 50,000.
 
-. Save the file to apply the changes. The limit is applied automatically.
+. Add the `enforcedLabelLimit`, `enforcedLabelNameLengthLimit`, and `enforcedLabelValueLengthLimit` configurations to `data/config.yaml` to limit the number of scraped labels, the length of the label name, and the length of a label value in user-defined projects:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    prometheus:
+      enforcedLabelLimit: 500 <1>
+      enforcedLabelNameLengthLimit: 50 <2>
+      enforcedLabelValueLengthLimit: 600 <3>
+----
+<1> Specifies the maximum number of labels per scrape. 
+The default value is `500`.
+A value of `0` specifies no limit.
+<2> Specifies the maximum length in characters of a label name.
+The default value is `50`.
+A value of `0` specifies no limit.
+<3> Specifies the maximum length in characters of a label value.
+The default value is `600`.
```

Comment:
default value is `0`, change to
The example value is `600`

Reply:
fixed